pkg/cri/server/helpers.go

```diff
@@ -517,3 +518,65 @@ func copyResourcesToStatus(spec *runtimespec.Spec, status containerstore.Status)
 	}
 	return status
 }
+
+func (c *criService) generateAndSendContainerEvent(ctx context.Context, containerID string, sandbox sandboxstore.Sandbox, eventType runtime.ContainerEventType) {
+	sandboxID := sandbox.ID
```

Comment:
check for nil sandbox and ignore if nil but don't allow the exception.. I think...

Reply:
done